lib/scala/tree/src/main/scala/io/github/tetherlessworld/twxplore/lib/tree/models/domain/NTA.scala

```diff
@@ -6,13 +6,13 @@ import io.github.tetherlessworld.twxplore.lib.base.models.domain._
 import org.apache.jena.rdf.model.{Model, Resource, ResourceFactory}
 
 //Nta
-final case class NTA(nta: String, ntaName: String, blocks: List[Uri], borough: Option[Uri], postCode: Uri) extends Ordered[NTA] {
+final case class NTA(nta: String, name: String, blocks: List[Uri], borough: Uri, postCode: Uri) extends Ordered[NTA] {
   val uri = Uri.parse("urn:treedata:resource:nta:" + nta)
 
   def compare(that: NTA) = this.nta compare that.nta
 
   def addBlock(block: Block): NTA = {
-    NTA(nta, ntaName, blocks :+ block.uri, borough, postCode)
```

Comment:
Since you have multiple Strings here (and elsewhere) you should probably err on the side of doing keyword arguments so there's no mixup. 

That said, you should be using .copy in any case.

Reply:
this.copy(blocks = blocks ++ block.uri) or similar